hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -1447,6 +1443,12 @@ protected void onClear() {
     }
 
     @Override
+    public void close() {
```

Comment:
Doesn't clearing records also requires invalidating entries (for near-cache) in this case? Because we are clearing entries. So you may call `clear()` inside this